Stripe/STPPaymentCardTextField.m

```diff
@@ -74,10 +77,12 @@ - (instancetype)initWithFrame:(CGRect)frame {
 }
 
 - (void)commonInit {
-    
-    self.borderColor = [self.class placeholderGrayColor];
-    self.cornerRadius = 5.0f;
-    self.borderWidth = 1.0f;
+    _borderColor = [self.class placeholderGrayColor];
+    _cornerRadius = 5.0f;
+    _borderWidth = 1.0f;
```

Comment:
I see. I think then probably you should add some comments explaining the issue in here so people dont delete these in a future refactor, but otherwise looks good.